openmc/deplete/microxs.py

```diff
@@ -89,7 +92,12 @@ def from_model(cls,
 
         # create temporary run
         with tempfile.TemporaryDirectory() as temp_dir:
-            statepoint_path = model.run(cwd=temp_dir)
+            if run_kwargs is None:
+                run_kwargs = {}
+                model.init_lib()
+            elif 'cwd' in run_kwargs:
+                run_kwargs.pop('cwd')
+            statepoint_path = model.run(cwd=temp_dir, **run_kwargs)
```

Comment:
I wouldn't put `init_lib()` within here. If a user wants to have things run through `openmc.lib`, they can call `model.init_lib()` themselves prior to calling `MicroXS.from_model`. Also, using the `setdefault` method on a dictionary can help clean this up:
```suggestion
            run_kwargs.setdefault('cwd', temp_dir)
            statepoint_path = model.run(**run_kwargs)
```